### PR TITLE
Support specifying the indent level when json dumping protobufs

### PR DESCRIPTION
**Pull request by mwildehahn**
I didn't keep the default of identing by 2 because I don't think its necessary.

### CODE REVIEW

**Comment by mhahn** [comment]
I signed it!

File: python/google/protobuf/json_format.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ class ParseError(Exception):
   """Thrown in case of parsing error."""
 
 
-def MessageToJson(message, including_default_value_fields=False):
+def MessageToJson(message, including_default_value_fields=False, indent=None):
```
Comment (xfxyjwf) [inline python/google/protobuf/json_format.py]: Please keep the original default value "2". Otherwise you will need to update the tests and that will be too much unnecessary changes.

Comment (mhahn) [inline python/google/protobuf/json_format.py]: sorry about that, I updated the tests and added a test case to ensure indentation is working when specified.

**Comment by xfxyjwf** [comment]
@anandolee to take a look.

**Comment by anandolee** [comment]
I am closing this PR because another change has already added  indent=2 in MessageToJson()
